### PR TITLE
Power8 rbconfig fixes

### PR DESCRIPTION
**Pull request by ScottGarman**
The previous rbconfig files included for Power8 platforms were badly broken; I had taken them from the packaged versions of ruby for the OS, instead of rebuilding from sources. This PR includes rbconfig files from a native compile from source I've performed on the target, which is what I should have done in the first place.

I've tested this by rebuilding the RHEL and Ubuntu agents with these fixes, and they now successfully show installed gems and I can require the gems using our irb command.

Additionally, I can install a gem with native extensions (sqlite3) with our gem command.

### CODE REVIEW

File: resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb [hunk 1]
```diff
@@ -196,8 +196,8 @@ module RbConfig
   CONFIG["build_vendor"] = "ibm"
   CONFIG["build_cpu"] = "powerpc64le"
   CONFIG["build"] = "powerpc64le-ibm-linux-gnu"
-  CONFIG["RUBY_RELEASE_DATE"] = "2016-03-30"
-  CONFIG["RUBY_PROGRAM_VERSION"] = "2.3.3"
+  CONFIG["RUBY_RELEASE_DATE"] = "2015-04-13"
+  CONFIG["RUBY_PROGRAM_VERSION"] = "2.1.9"
```
Comment (geoffnichols) [inline resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb]: Should we have the same` CONFIG["RUBY_PROGRAM_VERSION"]` in `rbconfig-ppc64le-redhat-linux.rb`?

File: resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb [hunk 2]
```diff
@@ -196,8 +196,8 @@ module RbConfig
   CONFIG["build_vendor"] = "ibm"
   CONFIG["build_cpu"] = "powerpc64le"
   CONFIG["build"] = "powerpc64le-ibm-linux-gnu"
-  CONFIG["RUBY_RELEASE_DATE"] = "2016-03-30"
-  CONFIG["RUBY_PROGRAM_VERSION"] = "2.3.3"
+  CONFIG["RUBY_RELEASE_DATE"] = "2015-04-13"
```
Comment (geoffnichols) [inline resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb]: I'm curious about this date. For our natively compiled Ruby 2.1.9, what is `CONFIG["RUBY_RELEASE_DATE"]`?

Comment (ScottGarman) [inline resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb]: I'm pretty sure this isn't relevant to fixing the error we were seeing, I did it to make things consistent with the other rbconfigs for this version.

**Comment by ScottGarman** [comment]
I think there may be a second issue to fix with installing gems with native extensions. Don't merge this just yet.

File: resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb (hunk 2 above)
Comment (geoffnichols) [inline resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb]: Ah, makes sense.

**Comment by ScottGarman** [comment]
This has now been heavily tested, including ensuring that our gem command works to build native extensions.

**Comment by McdonaldSeanp** [comment]
👍

**Pull request merged**